math/benches/msm.rs

```diff
@@ -0,0 +1,124 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use lambdaworks_math::cyclic_group::IsCyclicBilinearGroup;
+use lambdaworks_math::elliptic_curve::traits::HasDistortionMap;
+use lambdaworks_math::msm::naive::Naive;
+use lambdaworks_math::{
+    elliptic_curve::{
+        curves::test_curve::{QuadraticNonResidue, ORDER_P},
+        element::EllipticCurveElement,
+        traits::HasEllipticCurveOperations,
+    },
+    field::{
+        element::FieldElement, fields::u64_prime_field::U64PrimeField,
+        quadratic_extension::QuadraticExtensionField,
+    },
+    msm::{pippenger, MSM},
+};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+type FE = FieldElement<U64PrimeField<ORDER_P>>;
```

Comment:
It would be better if we could change this to something more descriptive like `FieldElement`
```suggestion
type FieldElement = FieldElement<U64PrimeField<ORDER_P>>;
```